velociraptor/autoplotter/lines.py

```diff
@@ -362,4 +362,9 @@ def plot_line(
                 linewidth=0.0,
             )
 
+        try:
+            ax.scatter(additional_x.value, additional_y.value, color=line.get_color())
+        except NameError:
```

Comment:
Why a `NameError` here? Not an `AttributeError` or something else?

Reply:
there is a case where `line` is undefined so we cannot get `line.get_color()`. This case is handled by `NameError`

Reply:
Okay, please add a comment describing that case.